Lib/fontTools/varLib/featureVars.py

```diff
@@ -298,6 +298,11 @@ def addFeatureVariationsRaw(font, conditionalSubstitutions, featureTag='rvrn'):
         varFeatureIndex = gsub.FeatureList.FeatureRecord.index(varFeature)
 
         for scriptRecord in gsub.ScriptList.ScriptRecord:
+            if scriptRecord.Script.DefaultLangSys is None:
```

Comment:
@madig @justvanrossum I wonder, instead of raising an error, we could simply skip the missing DefaultLangSys here?
I don't see the precise reason why it should be required.

Reply:
Well you can use it to force people to use `dflt`, which is good practice anyway I think?